orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/DetermineHealthProvidersTask.java

```diff
@@ -98,7 +105,11 @@ public TaskResult execute(Stage stage) {
 
     try {
       String applicationName = (String) stage.getContext().get("application");
-      Moniker moniker = (Moniker) stage.getContext().get("moniker");
+      Moniker moniker = null;
+      if (stage.getContext().containsKey("moniker")) {
+        moniker = objectMapper.convertValue(stage.getContext().get("moniker"), Moniker.class);
```

Comment:
The `stage.mapTo("/moniker", Moniker.class)` is more consistent with other uses IMO ... also does not require explicit injection of an `objectMapper`.

Any reason it didn't work for you?

Reply:
I didn't know it existed - will switch

Reply:
👍